drivers/soundwire/mipi_disco.c

```diff
@@ -280,6 +280,35 @@ static int sdw_slave_read_dpn(struct sdw_slave *slave,
 	return 0;
 }
 
+/*
+ * In MIPI DisCo spec for SoundWire, lane mapping for a slave device is done with
+ * mipi-sdw-lane-x-mapping properties, where x is 1..7, and the values for those
+ * properties are mipi-sdw-manager-lane-x or mipi-sdw-peripheral-link-y, where x
+ * is an integer between 1 to 7 if the lane is connected to a master lane, y is a
+ * character between A to E if the lane is connected to another slave lane.
```

Comment:
probably best to use manager and peripheral in a consistent manner. The last sentence should probably be 

"
where x is an integer between 1 to 7 if the lane is connected to a Manager, y is a character between A to E if the lane is connected to another Peripheral.
"

Reply:
Ack.